master/docs/relnotes/0.8.12.rst

```diff
@@ -95,6 +95,8 @@ Features
 * :class:`~buildbot.process.buildstep.Trigger`: the ``getSchedulersAndProperties`` customization method has been backported from Nine.
   This provides a way to dynamically specify which schedulers (and the properties for that scheduler) to trigger at runtime.
 
+* :bb:status:`GerritStatusPush` now has parameter --notify which is used to control e-mail notifications from Gerrit.
```

Comment:
Even though you submit a PR for 0.8.12, please update `docs/relnotes/index.rst` -- 0.8.12 is already released and its release notes are carved in stone :)